editions/tw5.com/tiddlers/plugins/Manually installing a plugin.tid

```diff
@@ -1,14 +1,30 @@
 created: 20160107222430613
-modified: 20190927205631648
+modified: 20220612082201459
 tags: Plugins
 title: Manually installing a plugin
 type: text/vnd.tiddlywiki
 
+\rules except wikilink
+\import Plugins
```

Comment:
What is this import pragma intended to do? It doesn't look like there are any definitions in the tiddler "Plugins"

Reply:
There are some macro definitions in the Plugins tiddler. The `<<open-edition-window` macro used in the following line is defined there. ... I thought they are very specific. BUT at a second look I think I can make them generic. So they will be available for the whole doc, to create a "side by side" view.